mzbench/README.md

```diff
@@ -55,4 +56,24 @@ In [fan-out](https://en.wikipedia.org/wiki/Fan-out) scenario one MQTT client is
 | MF_MZBENCH_MSG_SIZE  | Size of messages in bytes. Messages are a sequence of random bytes           | 100     |
 | MF_MZBENCH_QOS       | MQTT QoS level                                                               | 2       |
 
-
+## 1-to-1
+
+In 1-to-1 test numerous clients subscribe to an exclusive Mainflux channel and the same clients send messages on that channel to themselves.
```

Comment:
Why themselves? Is the same publisher at the same time subscriber?

I would like to avoid that - we would like to have a group of publishers and a group of subscribers, different clients. This will have lighter load on the clients and provide better metrics.

Reply:
Main reason is this is the way authors of MQTT worker we are using also did their 1-to-1 scenario:
https://github.com/vernemq/vmq_mzbench/blob/master/scenarios/1_to_1.bdl
Other reason is complications in scenario

Reply:
Approved, to stay aligned with VerneMQ team.